### DISCUSSION TITLE
氵

### DISCUSSION THREAD

**Discussion by Loxia1121** [share121/inter-knot#5198] [灌水]
有人知道为什么我已经登入Github了 为啥绳网还是显示未登入呢？

**Comment by Faixiang** [comment]
不知道，應該是延遲問題

**Comment by Dave-12138** [comment]
在github登陆后
- 回到绳网点设置
- 选择用户接口
- 回到绳网主界面，绳网提示需要github授权
- 根据提示完成授权
<img src="https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png" alt="流萤 也挺好" width="75" >

**Reply by Loxia1121** [reply]
好，我试试
